suite-native/message-system/src/components/MessageScreen.tsx

```diff
@@ -0,0 +1,132 @@
+import { Linking } from 'react-native';
+import { useDispatch } from 'react-redux';
+
+import { A, G } from '@mobily/ts-belt';
+
+import { IconName } from '@suite-common/icons/src';
+import { Box, Button, Pictogram, PictogramVariant, VStack } from '@suite-native/atoms';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { Message, Variant } from '@suite-common/suite-types';
+import { messageSystemActions } from '@suite-common/message-system';
+import { useTranslate } from '@suite-native/intl';
+import { Screen } from '@suite-native/navigation';
+
+type MessageScreenProps = {
+    message: Message;
+};
+
+const screenStyle = prepareNativeStyle(utils => ({
+    justifyContent: 'center',
+    alignItems: 'center',
+    paddingHorizontal: utils.spacings.large,
+    marginBottom: utils.spacings.extraLarge,
+    flexGrow: 1,
+}));
+
+const buttonsWrapperStyle = prepareNativeStyle(utils => ({
+    position: 'absolute',
+    left: 0,
+    right: 0,
+    bottom: 0,
+    paddingHorizontal: utils.spacings.large,
+    marginBottom: utils.spacings.extraLarge,
```

Comment:
Since screen has been applied, wouldn't it be possible to position this with flex instead of absolute?

Reply:
ab1bdabbd